packages/website/utils/content.ts

```diff
@@ -83,11 +83,19 @@ async function getMdxPaths() {
 
   // creates a json with a list of links grouped into sections
   const sidebarLinks = pages.reduce((acc, page) => {
-    const { title, slug, section, isNew = undefined } = page.frontMatter.data;
+    const {
+      title,
+      slug,
+      section,
+      isNew = undefined,
+      isBeta = undefined,
+      isAlpha = undefined,
+      isDeprecated = undefined,
```

Comment:
why do we need `undefined` by default here?

Reply:
Good question! 🙂 I didn’t investigate much here, just added new values on top of the ligic that was already in place. I can check it out.